discord/ext/menus/__init__.py

```diff
@@ -898,17 +984,18 @@ class MenuPages(Menu):
         The current page that we are in. Zero-indexed
         between [0, :attr:`PageSource.max_pages`).
     """
-    def __init__(self, source, **kwargs):
+
+    def __init__(self, source: PageSource[T], **kwargs: Any):
```

Comment:
This is also missing a return type
```suggestion
    def __init__(self, source: PageSource[T], **kwargs: Any) -> None:
```